### DISCUSSION TITLE
paddleocr export_model failed!

### DISCUSSION THREAD

**Discussion by OeasyC** [PaddlePaddle/PaddleOCR#13950] [Q&A]
paddleocr export_model failed!
![image](https://github.com/user-attachments/assets/187544ed-c806-484d-a111-79547404f032)

env:
![image](https://github.com/user-attachments/assets/7cf13d35-1cda-4a5e-afb5-003a39f4b8da)
![image](https://github.com/user-attachments/assets/1da17804-83f7-4b1a-80b3-e8881aedaaf8)

device:
3090*2

**Comment by UserWangZz** [comment]
The above information is not enough to determine the problem

**Reply by OeasyC** [reply]
> The above information is not enough to determine the problem
这是报错的具体位置，和详细信息：
使用yml是：en_PP-OCRv3_rec.yml  看起来像是默认的配置文件和导出代码有冲突，配置文件中为SVRT+mutilHead，而代码中在这个配置下input_shape=None，这好像导致了后续的导出错误。请帮忙看一下，十分感谢。❤
![image](https://github.com/user-attachments/assets/79643535-0a81-452d-bfda-7fbf41f8b752)

![image](https://github.com/user-attachments/assets/25643189-c8a4-4e77-ad1c-5a9dc5b4ea0e)
![image](https://github.com/user-attachments/assets/740b4cda-3423-4419-9f3a-0fff6c40a844)
